pm2.json

```diff
@@ -1,8 +1,10 @@
 {
   "apps": [
     {
-      "name": "edge-sync-server",
-      "script": "lib/index.js"
+      "name": "server",
```

Comment:
You don't want to also run the "mon.database" and the "mon.network"?
I'm genuinely asking, I have no idea if it makes sense.

Reply:
I hadn't thought about it. Now that you mentioned it, I'll have to think if that makes sense. Regardless, I think that could be a separate task and PR.